docs/data-sources/registry_namespace_beta.md

```diff
@@ -31,9 +31,9 @@ data "scaleway_registry_namespace_beta" "my_namespace" {
 - `namespace_id` - (Optional) The namespace id.
   Only one of `name` and `namespace_id` should be specified.
 
-- `region` - (Defaults to [provider](../index.html#region) `region`) The [region](../guides/regions_and_zones.html#regions) in which the namespace exists.
+- `region` - (Defaults to [provider](../index.md#region) `region`) The [region](../guides/regions_and_zones.md#regions) in which the namespace exists.
 
-- `organization_id` - (Defaults to [provider](../index.html#organization_id) `organization_id`) The ID of the organization the namespace is associated with.
+- `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the namespace is associated with.
```

Comment:
`html` replace by `md ` will it work on terraform website ?

Reply:
According to https://registry.terraform.io/tools/doc-preview it will remove `.md`